plugins/jobtype/build.xml

```diff
@@ -36,7 +36,7 @@
 
 		<pathelement path="${dist.classes.dir}" />
 	</path>
-	
+
```

Comment:
We should also document it after this pull request to allow open source to leverage this work.
http://azkaban.github.io/azkaban/docs/latest